examples/platform/silabs/SiWx917/SiWx917/sl_wifi_if.c

```diff
@@ -307,7 +310,17 @@ int32_t wfx_wifi_rsi_init(void)
     if (status != SL_STATUS_OK)
     {
         SILABS_LOG("wfx_wifi_rsi_init failed %x", status);
+        return status;
     }
+
+    // Create Sempaphore for scan
+    sScanSemaphore = osSemaphoreNew(1, 0, NULL);
+    if (sScanSemaphore == NULL)
+    {
+        SILABS_LOG("Failed to create scan semaphore");
```

Comment:
I would remove all of the logs from this function and instead add one in the MatterConfig.cpp. For clarity and code size.